### ISSUE TITLE
Include minification note

### ISSUE THREAD

**Issue by gaboesquivel** [feross/standard#278]
It believe it is important to include a note ( perhaps under FAQ ) on relation to minification. Mention that [UglifyJS2](https://github.com/mishoo/UglifyJS2) will safely minify the javascript since it does ASI by default.

**Comment by rstacruz** [comment]
are there known minifiers that don't work? I'm under the assumption here that most production-grade minifiers today should be able to handle semicolon-less JS.

**Comment by gaboesquivel** [comment]
@rstacruz I've always used semicolons and I never give it a thought to ASI support in minifiers since I'm used to lint before minification. When I decided to try to this style it was one of my first concerns, I had to read the minifier docs to realize it's all good. Perhaps this knowledge shouldn't be assumed... someone could that have the same concern and finding a note in standard would be useful. As you phrase would suffice _"most production-grade minifiers today should be able to handle semicolon-less JS."_
